pkg/registry/apis/provisioning/register.go

```diff
@@ -71,6 +71,9 @@ var (
 	_ builder.APIGroupRouteProvider         = (*APIBuilder)(nil)
 	_ builder.APIGroupPostStartHookProvider = (*APIBuilder)(nil)
 	_ builder.OpenAPIPostProcessor          = (*APIBuilder)(nil)
+
+	// For now max 15min jobs
+	jobTimeout = 15 * time.Minute
```

Comment:
this was referenced by both the driver and the job store -- so I set it as a variable

Reply:
but it isn't now, right?